src/lib/err/backtrace.c

```diff
@@ -190,13 +191,15 @@ dump_stack_symbols_to_error_fds(void)
     backtrace_symbols_fd(cb_buf, (int)depth, fds[i]);
 }
 
+/* The signals that we want our backtrace handler to trap */
+static int trap_signals[] = { SIGSEGV, SIGILL, SIGFPE, SIGBUS, SIGSYS,
+  SIGIO, -1 };
```

Comment:
This in theory should be indented properly to follow our coding standards. I'm not going to hold up on this though to get it merged but if you have a minute, great! :)